.github/workflows/docker.yml

```
@@ -52,28 +50,44 @@ jobs:
       - name: Run brew test-bot --only-setup
         run: docker run --rm brew brew test-bot --only-setup
 
+      - name: Log in to GitHub Packages
+        if: >
+          github.event_name == 'release' ||
+          (github.event_name == 'push' && github.ref == 'refs/heads/master')
+        run: |
+          echo "$TOKEN" | docker login ghcr.io -u BrewTestBot --password-stdin
```

```suggestion
          docker login ghcr.io -u BrewTestBot --password-stdin <<<"$TOKEN"
```
This makes it slightly more obvious what's happening if you're reading from left to right.

Only a mild preference, though. Feel free to ignore.